static/app/views/insights/sessions/charts/userHealthCountChart.tsx

```diff
@@ -15,6 +16,7 @@ export default function UserHealthCountChart() {
 
   return (
     <InsightsLineChartWidget
+      height={SESSION_HEALTH_CHART_HEIGHT}
```

Comment:
looks like every chart has the same height 👍 

this'll make it easy to put any chart in any position per https://github.com/getsentry/sentry/pull/88502
also, we'll get a merge conflict from editing similar lines, but it'll be easy to resolve. 
the resolution will be about the same in all cases:
```
      height={SESSION_HEALTH_CHART_HEIGHT}
      title={CHART_TITLES.zzzzz}
      interactiveTitle={() => (
        <ChartSelectionTitle title={CHART_TITLES.zzzzz} />
      )}
```